pkg/types/v1beta1/constants.go

```diff
@@ -21,8 +21,9 @@ const (
 )
 
 var (
-	MASTER = "master"
-	NODE   = "node"
+	MASTER   = "master"
+	NODE     = "node"
+	REGISTRY = "registry"
```

Comment:
We can, but not necessary. if not registry role assigned, the master0 will be the default one.

Reply:
> We can, but not necessary. if not registry role assigned, the master0 will be the default one.

It is default, but it is best for the configuration file to be tracked, otherwise it may be different from the logic of the application.